radicle/src/cob/patch.rs

```diff
@@ -1042,6 +1042,38 @@ mod test {
         assert_eq!(revision.base, base);
     }
 
+    #[test]
+    fn test_patch_discussion() {
+        let tmp = tempfile::tempdir().unwrap();
+        let (_, signer, project) = test::setup::context(&tmp);
+        let mut patches = Patches::open(*signer.public_key(), &project).unwrap();
+        let patch = patches
+            .create(
+                "My first patch",
+                "Blah blah blah.",
+                MergeTarget::Delegates,
+                git::Oid::try_from("cb18e95ada2bb38aadd8e6cef0963ce37a87add3").unwrap(),
+                git::Oid::try_from("e2a85016a458cd809c0ecee81f8c99613b0b0945").unwrap(),
+                &[],
+                &signer,
+            )
+            .unwrap();
+
+        let id = patch.id;
+        let mut patch = patches.get_mut(&id).unwrap();
+        let (revision_id, _) = patch.revisions().last().expect("patch has a revision");
+        assert!(
+            patch
+                .comment(*revision_id, "patch comment", None, &signer)
+                .is_ok(),
+            "can comment on patch"
+        );
+
+        let (_, revision) = patch.revisions().last().expect("patch has a revision");
+        let (_, comment) = revision.discussion.first().expect("comment saved");
```

Comment:
We generally use `unwrap` in tests, as the code is too noisy otherwise, and it's enough to point to the line that panicked.

Reply:
Noted :+1: